src/modules/Inquiries.js

```diff
@@ -21,3 +21,14 @@ const Inquiries = {
 }
 
 export default Inquiries
+
+const setLanguageValue = () => {
+  switch (i18n.language) {
+    case 'English':
+      return 'en'
+    case 'Svenska':
+      return 'se'
+    default:
+      return 'en'
```

Comment:
Weren’t we gonna have Swedish as default?

Reply:
Let's change it here when we make the whole site Swedish default

Reply:
oki now i see (y)